test/parallel/test-benchmark-querystring.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+require('../common');
+
+const runBenchmark = require('../common/benchmark');
+
+runBenchmark('querystring', ['n=1', 'input=', 'type=']);
```

Comment:
I believe `n=1` is probably enough. Setting `type=` would leave some codepaths uncovered.

Reply:
I've added a `type` and `input` from the regular benchmark run. This is my first time working with the node.js benchmarks; thanks for reviewing and your help.